pkg/function/scale_workload_test.go

```diff
@@ -0,0 +1,46 @@
+package function
```

Comment:
This file needs the license header

Reply:
I will make sure that I add this as part of one of the next PRs.